### PR TITLE
Timeout added while removing worker node

### LINKED ISSUE #269: Terraform destroy fails when evicting pods error out as no workers left
Evict pod sometimes take longer with below error. This happens because there are no workers left and the pod cannot be allocated.

```
module.nodes.null_resource.remove_worker[0]: Still destroying... [id=2071292489472051445, 1m50s elapsed]
module.nodes.null_resource.remove_worker[0] (remote-exec): evicting pod openshift-ingress/router-default-b576fd579-tvx48                      
module.nodes.null_resource.remove_worker[0] (remote-exec): error when evicting pods/"router-default-b576fd579-tvx48" -n "openshift-ingress" (will retry after 5s): Cannot evict pod as it would violate the pod's disruption budget.
module.nodes.null_resource.remove_worker[0] (remote-exec): evicting pod openshift-ingress/router-default-b576fd579-tvx48
module.nodes.null_resource.remove_worker[0] (remote-exec): error when evicting pods/"router-default-b576fd579-tvx48" -n "openshift-ingress" (will retry after 5s): Cannot evict pod as it would violate the pod's disruption budget.
```

### PR DESCRIPTION
**Pull request by yussufsh**
Added 3 min wait for the drain command to complete. This will help when no worker nodes are left for scheduling non disruptive pods.

Fixes #269

Signed-off-by: Yussuf Shaikh <yussuf.shaikh@ibm.com>

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/270#" title="Author self-approved">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by cs-zhang — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/270#pullrequestreview-648262211" title="LGTM">cs-zhang</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/270#" title="Author self-approved">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by Prajyot-Parab — APPROVED** [review]
/approve

**Comment by ltccci** [comment]
@yussufsh: new pull request created: #273

<details>

In response to [this](https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/270#issuecomment-837796602):

>/cherrypick release-4.7


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by ltccci** [comment]
@yussufsh: new pull request created: #274

<details>

In response to [this](https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/270#issuecomment-837797009):

>/cherrypick release-4.6


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**
